packages/actor-rdf-resolve-hypermedia-qpf/test/RdfSourceQpf-test.ts

```diff
@@ -4,6 +4,7 @@ import type { IActorDereferenceRdfOutput } from '@comunica/bus-dereference-rdf';
 import { ActionContext, Bus } from '@comunica/core';
 import type * as RDF from '@rdfjs/types';
 import arrayifyStream from 'arrayify-stream';
+import { DataFactory as N3DF } from 'n3';
```

Comment:
Is there a reason why another datafactory is used here?

Reply:
Not  now - it was an artefact of me thinking that rdf-data-factory was setting the value of the default graph term to be "DEFAULT_GRAPH"

Reply:
It also uses the empty string if I remember correctly. I guess we can keep using the usual factory here then?